clients/da-vinci-client/src/main/java/com/linkedin/davinci/ingestion/IsolatedIngestionBackend.java

```diff
@@ -248,6 +252,28 @@ public void prepareForShutdown() {
     isShuttingDown.set(true);
   }
 
+  @Override
+  public Map<String, Set<Integer>> getLoadedStoreUserPartitionsMapping() {
+    /**
+     * Merge the loaded store partitions from main process and isolated process.
+     */
+    Map<String, Set<Integer>> localMapping = super.getLoadedStoreUserPartitionsMapping();
+    LoadedStoreUserPartitionMapping isolatedMapping = mainIngestionRequestClient.getLoadedStoreUserPartitionMapping();
+    LOGGER.info(
+        "Got loaded store partition mapping from isolated process: {}",
+        isolatedMapping.storeUserPartitionMapping);
```

Comment:
nit: For prod server, this will be a very big log line containing thousands of mapping... it is not going to look good though. But probably this is what we can get now as trimming some of them is also useless....

Reply:
yes, there will be one list per store version, and I feel it is useful to track this behavior since II communication is tricky and there is no read traffic at start/stop time, so the impact to the online traffic is none and it will pollute the log a little bit and we can evaluate it later.